code/tools/make_munki_mpkg.sh

```diff
@@ -490,7 +496,7 @@ chmod -R go-w "$APPROOT/Applications/Managed Software Center.app"
 # sign MSC app
 if [ "$APPSIGNINGCERT" != "" ]; then
     echo "Signing Managed Software Center.app..."
-    /usr/bin/codesign -f -s "$APPSIGNINGCERT" --options runtime --verbose \
```

Comment:
Why would we need to --force a signature on a newly-built app? In any case this would be unrelated to adding a new option to do an auto run. Please revert this.